doc/dftb+/manual/dftbp.tex

```diff
@@ -2883,6 +2881,46 @@ \subsubsection{DftD4 ChargeModel}
 }
```

Comment:
I'm a bit concerned about speed and readability here. How fast is building the input when I also allow modifiers? Also I don't want to overload the input by attaching modifiers to each species symbol. The alternative to attach the modifier to the parent turned out to be fragile in case we use a hybrid of defaults and user values.

Reply:
The unit of the chemical hardness should be energy per square-charge, because it is the second derivative of the energy w.r.t. the population. Not sure if you really want to have units on that one.